src/npctalk.cpp

```diff
@@ -2325,6 +2326,14 @@ void talk_effect_fun_t::set_consume_item( const JsonObject &jo, const std::strin
         if( is_npc ) {
             consume_item( *d.actor( true ), item_name, count, charges );
         } else {
+            if( do_popup ) {
+                if( count == 1 ) {
+                    popup( _( "You give %1$s a %2$s." ), d.actor( true )->disp_name(), item::nname( item_name ) );
+                } else {
+                    popup( _( "You give %1$s %2$d %3$s." ), d.actor( true )->disp_name(), count,
```

Comment:
```suggestion
                if( count == 1 ) {
                    //~ 1$ = character's name, 2$ = item name
                    popup( _( "You give %1$s a %2$s." ), d.actor( true )->disp_name(), item::nname( item_name ) );
                } else {
                    //~ 1$ = character's name, 2$ = number of items, 3$ = item name
                    popup( _( "You give %1$s %2$d %3$s." ), d.actor( true )->disp_name(), count,
```
Might be helpful to leave a note for the translators

Reply:
Oh wait, sorry, it's already merged! :P